src/platformdirs/windows.py

```diff
@@ -80,28 +92,45 @@ def user_log_dir(self) -> str:
             path = os.path.join(path, "Logs")
         return path
 
+    @property
+    def user_documents_dir(self) -> str:
+        """
+        :return: documents directory tied to the user e.g. ``%USERPROFILE%\\Documents``
+        """
+        path = get_win_folder("CSIDL_PERSONAL")
+        if path:
+            path = os.path.normpath(path)
+        else:
+            path = os.path.join(os.path.normpath(os.environ["USERPROFILE"]), "Documents")
+        return path
+
     @property
     def user_runtime_dir(self) -> str:
         """
         :return: runtime directory tied to the user, e.g.
          ``%USERPROFILE%\\AppData\\Local\\Temp\\$appauthor\\$appname``
         """
-        path = os.path.normpath(os.path.join(get_win_folder("CSIDL_LOCAL_APPDATA"), "Temp"))
+        path = get_win_folder("CSIDL_LOCAL_APPDATA")
+        if path is None:
+            raise ValueError("No path found for CSIDL: CSIDL_LOCAL_APPDATA")
+
+        path = os.path.normpath(os.path.join(path, "Temp"))
         return self._append_parts(path)
 
 
-def get_win_folder_from_env_vars(csidl_name: str) -> str:
+def get_win_folder_from_env_vars(csidl_name: str) -> Optional[str]:
     """Get folder from environment variables."""
     env_var_name = {
         "CSIDL_APPDATA": "APPDATA",
         "CSIDL_COMMON_APPDATA": "ALLUSERSPROFILE",
         "CSIDL_LOCAL_APPDATA": "LOCALAPPDATA",
     }.get(csidl_name)
     if env_var_name is None:
-        raise ValueError(f"Unknown CSIDL name: {csidl_name}")
+        return None
```

Comment:
Let's restore the ValueErrors here (and remove the None checks from the other methods). Considering only user_documents_dir can raise here it's fine to catch the error in that case and fallback.